crates/pep508-rs/src/lib.rs

```diff
@@ -617,9 +617,48 @@ fn parse_extras(cursor: &mut Cursor) -> Result<Vec<ExtraName>, Pep508Error> {
     let Some(bracket_pos) = cursor.eat_char('[') else {
         return Ok(vec![]);
     };
+    cursor.eat_whitespace();
```

Comment:
Moved this out from the start of the loop, because it's also at the end of the loop.